src/patternfly/components/Expandable/examples/expandable-expanded-example.hbs

```diff
@@ -0,0 +1,15 @@
+{{#> expandable expandable--IsExpanded="true"}}
+  {{#> expandable-section}}
+    This section is always visible.
+  {{/expandable-section}}
+  {{#> expandable-toggle expandable-toggle--attribute='aria-controls="expanded-content-1" aria-expanded="true"'}}
```

Comment:
nitpicky but can you change the ID for this one to be "2" since it's the 2nd example, and change the ID for the non-expanded example to be "1"?